lib/web_ui/lib/src/engine/houdini_canvas.dart

```diff
@@ -22,7 +22,7 @@ class HoudiniCanvas extends EngineCanvas with SaveElementStackTracking {
   /// where this canvas paints.
   ///
   /// Painting outside the bounds of this rectangle is cropped.
-  final ui.Rect bounds;
+  final ui.Rect? bounds;
```

Comment:
why nullable?

Reply:
Should we kill `HoudiniCanvas` until we're ready for a proper investigation? Not sure if it's worth maintaining it with no tests and users.

Reply:
Please go ahead.